etc/nsjail/sandbox.cfg

```diff
@@ -28,7 +28,7 @@ cgroup_net_cls_parent: "ce-sandbox"
 cgroup_cpu_parent: "ce-sandbox"
 
 cgroup_mem_max: 209715200 # 200 MiB
-cgroup_pids_max: 4
+cgroup_pids_max: 14
```

Comment:
This is the minimum amount of pids I was able to make the executors run. Started discussion on CE's Discord regarding this.

Reply:
Thanks for commenting this. It hurts me this isn't a power of two ;) We need to work out on our side if having multiple executions, each having 14 (or 16 😁 ) threads is a DoS potential. I think _not_ as the `cgroup_cpu_ms_per_sec` should limit complete CPU takeover.